### DISCUSSION TITLE
Did a 51% attack already occured?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#5875] [General]
Since hpool reached a 50%+ network capacity a couple of weeks ago and there is a superficial growth in the netspace is it possible that they have already attacked the network?

**Comment by lvcivs** [comment]
There is a big difference between the network share that they report and the one that we see in the rewards going out. The latter has been pretty stable around 35% recently. So most likely the numbers on their website are wrong.
